app_bikes/forms.py

```diff
@@ -0,0 +1,26 @@
+from dal import autocomplete
+from django import forms
+
+
+class BikeModelForm(forms.ModelForm):
+    def __init__(self, *args, **kw):
+        super(BikeModelForm, self).__init__(*args, **kw)
+        if self.instance is not None:
+            # Saved instance is loaded, setup choices to display the selected value
+            self.fields['id_station'].widget.choices = ((self.instance.id_station, self.instance.station),)
```

Comment:
ok, I think I got it. I haven't checked into the autocompletion library code/doc, so dunno if there's an easier way to do that. It does seem a bit hacky though (and doesn't seem to work for me at least, as I commented, editing a configuration doesn't pre-fill the autocomplete field).